src/prelude.rs

```diff
@@ -402,6 +402,12 @@ pub fn parse_paths(
             .replace("<regHkcu>", SKIP)
             .replace("<regHklm>", SKIP),
     );
+    if root.store == Store::GogLinux {
```

Comment:
Let's change this to `if root.store == Store::Gog && get_os() == Os::Linux` and remove the `GogLinux` variant.